src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+  <React.StrictMode>
```

Comment:
react strict mode can cause you problems. It reloads too much. Look at how the lesson in the kernel told you to setup your index.js file